src/tenderly/tenderly.config.ts

```diff
@@ -57,6 +57,7 @@ export const TENDERLY_CHAINS: number[] = [
   ChainId.INK, // Ink - 57073
   ChainId.SWL, // Swellchain - 1923
   ChainId.CRN, // Corn - 21000000
+  ChainId.BOC, // BOB - 60808
```

Comment:
can you please also add other missing new chains? 😇

Reply:
The other chains I added are all not supported by tenderly (XDC & HyperEVM)